src/config.js

```diff
@@ -256,7 +257,7 @@ export default class Config {
     this.modulesFolder = opts.modulesFolder;
     this.globalFolder = opts.globalFolder || constants.GLOBAL_MODULE_DIRECTORY;
     this.linkFolder = opts.linkFolder || constants.LINK_REGISTRY_DIRECTORY;
-    this.production = !!opts.production;
+    this.production = !!opts.production && opts.production !== 'false';
```

Comment:
Just a nit, is this this.production assignment needed?
Maybe it should set production to false?